src/components/UI/Dropdown/index.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import i18next from "i18next";
+// Styles
+import { Container } from "./dropdown.styled";
+
+export function Dropdown() {
+  const handleLanguageChange = (lang = "en") => {
+    i18next.changeLanguage("sr");
```

Comment:
Shouldn't this change to `lang` instead of being hard coded?

Reply:
Yes it should, i left it so we can have different task for creating reusable dropdown component separately